### PR TITLE
Feature: Save Logs to pinchito.com

### PR DESCRIPTION
**Pull request by risdevs**
Restored the highly awaited feature of saving logs to pinchito.com

The feature uses a `map[tgId] Data` to keep track of the currently opened logs (each user can only edit one log at a time)

The steps to save a log are
```
1. /log
1. < forward messages >
1. /end_log
1. < add title >
1. < add author >
1. < confirm >
```

There are a couple of utility commands too
`/start` to welcome the user
`/cancel` to cancel the currently editing log for that user

@brafales I modified `main.go` to use polling instead of webhooks. Can you move my code back to webhooks? (I'm unable to test it locally)

### CODE REVIEW

File: handler/crear.go [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+		pinLog.Text += line
+	}
+	// TODO Handle other types of messages
+	// (Audio, img, ...)
+
+	return nil
+}
+
+func (t *Crear) handleKeyboardReply(update tgbotapi.Update) error {
+	host := os.Getenv("PINCHITO_HOST")
```
Comment (brafales) [inline handler/crear.go]: Can we move this into an attribute of `Crear` and inject it in when we create the handler?

File: pinchito/client.go [hunk 2]
```diff
@@ -4,11 +4,15 @@ import (
 	"encoding/json"
 	"net/http"
 	"net/url"
+	"errors"
+	"bytes"
+	"os"
 )
 
 func Tapeta() (Log, error) {
+	host := os.Getenv("PINCHITO_HOST")
```
Comment (brafales) [inline pinchito/client.go]: I'd do the same here. Maybe make these methods part of the `Client` type and pass in the `PINCHITO_HOST` as a value when initialising it

File: pinchito/users.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+type TgPinchitoUser struct {
+	TgUsername  string
+	PinId int
+	PinNick  string
+}
+
+var tgPinchitoUsers []TgPinchitoUser
+
+func InitUsers() {
+	tgPinchitoUsers = []TgPinchitoUser{
```
Comment (brafales) [inline pinchito/users.go]: I'd like to explore the possibility of loading this as a JSON config file rather than having it here, publicly available

**Review by brafales — CHANGES_REQUESTED** [review]
Can we ensure we've run `go fmt` as well? I'd like to see some tests and I have the feeling the `Crear` handle is too big, but we can do some refactor later on.

File: pinchito/users.go (hunk 3 above)
Comment (risdevs) [reply pinchito/users.go]: where would you put this file?
and how would you load it?

**Review by brafales — APPROVED** [review]

**Pull request merged**